src/broker/cas_cgw.c

```diff
@@ -40,6 +40,18 @@
                                 } \
                             }
 
+#define SUPPORTED_DBMS_MAX_NUM     2
+
+typedef struct t_supported_dbms T_SUPPORTED_DBMS;
+struct t_supported_dbms
+{
+  char *dbms_name;
+  SUPPORTED_DBMS_TYPE dbms_type;
+};
+
+static T_SUPPORTED_DBMS supported_dbms_list[SUPPORTED_DBMS_MAX_NUM] =
+  { {"oracle", SUPPORTED_DBMS_ORACLE}, {"mysql", SUPPORTED_DBMS_MYSQL} };
```

Comment:
propose:
remove #43

`#define SUPPORTED_DBMS_MAX_NUM     2`

```
static T_SUPPORTED_DBMS supported_dbms_list[] = {
  {"oracle", SUPPORTED_DBMS_ORACLE}, 
  {"mysql", SUPPORTED_DBMS_MYSQL}
};

int supported_dbms_max_num = sizeof (supported_dbms_list) / sizeof (T_SUPPORTED_DBMS);
```

Reply:
Thank you for your comments.